python/pyspark/pandas/tests/test_ops_on_diff_frames.py

```diff
@@ -1955,6 +1955,28 @@ def test_pow_and_rpow(self):
         with self.assertRaisesRegex(ValueError, "Cannot combine the series or dataframe"):
             psser.rpow(psser_other)
 
+    def test_cov(self):
+        from pyspark.pandas.config import set_option, reset_option
+
+        set_option("compute.ops_on_diff_frames", True)
```

Comment:
We may not need to set the option in this class.

Reply:
It will be set https://github.com/apache/spark/blob/f0d8cd515e6ec440b67e612e056e42c15513a349/python/pyspark/pandas/tests/test_ops_on_diff_frames.py#L42.

Reply:
thank you, updated.